src/main/resources/db/migration/V23__revise_court_ids.sql

```diff
@@ -0,0 +1,11 @@
+delete from enabled_court where id = 'DRBYMC';
+
+update enabled_court set id = 'DRBYMC' where id = 'DRBYJC';
+
```

Comment:
All court_id will be null before this update.
Some will not be null afterwards.  
On preprod data this catches all but 9 of the video_link_appointment where made_by_court is true. (5 bookings).